templates/repository/conf/user-mgt.xml.erb

```diff
@@ -1,30 +1,28 @@
-
 <!--
-  ~  Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+  ~ Copyright WSO2, Inc. (http://wso2.com)
```

Comment:
update license.

Reply:
Not needed since only some of the configs where updated.